src/completions/Tab.ts

```diff
@@ -43,9 +43,7 @@ class BufferCompletionOption
         const favIconUrl = tab.favIconUrl
             ? tab.favIconUrl
             : Completions.DEFAULT_FAVICON
-        const indicator = tab.audible
-            ? String.fromCodePoint(0x1F50A)
-            : ""
+        // const indicator = tab.audible ? String.fromCodePoint(0x1f50a) : ""
```

Comment:
I don't see any good reason why this was removed, or why the code point became lowercase (if that even matters?)